app/uk/gov/hmrc/propertylinking/controllers/valuations/ValuationsController.scala

```diff
@@ -137,7 +158,9 @@ class ValuationsController @Inject()(
       clientDetails match {
         case Some(client) =>
           config.newDashboardUrl(
-            s"selected-client-properties?clientOrganisationId=${client.organisationId}&clientName=${client.organisationName}&pageNumber=1&pageSize=15&sortField=ADDRESS&sortOrder=ASC")
+            s"selected-client-properties?clientOrganisationId=${client.organisationId}&clientName=${URLEncoder
+              .encode(client.organisationName, "UTF-8")}&pageNumber=1&pageSize=15&sortField=ADDRESS&sortOrder=ASC")
```

Comment:
are these the defaults (&pageNumber=1&pageSize=15&sortField=ADDRESS&sortOrder=ASC)
if so, we don't need them here.
I'm not sure if they are, but might simplify that code

Reply:
yes they are defaults but unfortunately the link doesn't seem to work without them 🤷‍♂️ . That is the only reason why I added them